bootstrap/common/sync_third_party.sh

```diff
@@ -55,7 +55,8 @@ function sync_third_party() {
 
   # Install a trap to remove temporary path if this script is finished.
   # The trap will work well when unknown breaking happens like ctrl+c.
-  trap "rm -rf $temp_path" EXIT
+  export TRAP_COMMAND="rm -rf $temp_path;$TRAP_COMMAND"
```

Comment:
Why do we need to export this command?
Also, shouldn't be |export TRAP_COMMAND="rm -rf $temp_path"|?